examples/webgl_camera_array.html

```diff
@@ -78,6 +78,12 @@
 				const geometryCylinder = new THREE.CylinderGeometry( 0.5, 0.5, 1, 32 );
 				const materialCylinder = new THREE.MeshPhongMaterial( { color: 0xff0000 } );
 
+				for (let i = 0; i < mesh.count; i++) {
+				    const color = new THREE.Color(Math.random(), Math.random(), Math.random()); // Generates a random color
+				    mesh.setColorAt(i, color);
+				}
+				mesh.instanceColor.needsUpdate = true; // Ensure the color update is applied
```

Comment:
Sweep: This is the correct file, but the code does not compile so it has failed the CI. You do not need to update the color of the vertices. Instead can you just update the material of the cylinder to a bright yellow?